src/Message.php

```diff
@@ -151,6 +152,7 @@ public function setTimeToLive($value)
         return $this;
     }
 
+    #[ReturnTypeWillChange]
```

Comment:
Let's use  #[\ReturnTypeWillChange] instead so that we are not required ti import the namespace and cause errors on php versions lower then 8.1 for not finding it.

Reply:
Complete